tests/nnfw_api/src/GenModelTests/nontrainable_op_trains/Relu6.test.cc

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "GenModelTrain.h"
+
+TEST_F(GenModelTrain, NonTrainableOps_FC_Relu6_FC)
+{
+  // (( Input 0 )) -> [ FC ] -> [ Relu6 ] -> [ FC ] -> (( Output 0 ))
```

Comment:
If `(( Input 0 )) -> [ FC ] -> [ Relu6 ] -> (( Output 0 ))`, loss value does not changed. So I added one more FC layer.